features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -327,11 +334,56 @@ class Comment extends React.Component {
                                             allComments={allComments}
                                         />
                                     )} */}
+                                        {comment.replies && comment.replies.list.slice(0).map((replie, index) => (
+                                            <>
+                                                <Box ml={8}>
+                                                    {index !== 0 && <Divider className={classes.divider} />}
+                                                    <Grid container spacing={1} className={classes.root}>
+                                                        <Grid item>
+                                                            <Icon className={classes.commentIcon}>account_circle</Icon>
+                                                        </Grid>
+                                                        <Grid item xs zeroMinWidth>
+                                                            <Typography noWrap className={classes.commentText}>
+                                                                {(replie.commenterInfo && replie.commenterInfo.fullName)
```

Comment:
shall we change variable name **replie** to **reply**

Reply:
Fixed